### PR TITLE
feat(table): make file table sortable

### PR DESCRIPTION
**Pull request by safeamiiir**
to make the list of files in the experimentDetails page sortable.

### CODE REVIEW

File: aqueductcore/frontend/src/helper/functions.tsx [hunk 1]
```diff
@@ -15,3 +19,53 @@ export const focusInCurrentTarget = ({
...
+  if (itemB < itemA) {
+    return -1;
+  }
+  if (itemB > itemA) {
+    return 1;
+  }
+  return 0;
+}
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (samiralavi) [inline aqueductcore/frontend/src/helper/functions.tsx:44]: Why linting is disabled on these functions?

**Review by samiralavi — APPROVED** [review]
Thanks @safeamiiir. The PR looks, good. I assume there is a default sorting, either by name or by date when the experiment page is loaded.

File: aqueductcore/frontend/src/helper/functions.tsx (hunk 1 above)
Comment (safeamiiir) [reply aqueductcore/frontend/src/helper/functions.tsx]: fixed

**Comment by safeamiiir** [comment]
> Thanks @safeamiiir. The PR looks, good. I assume there is a default sorting, either by name or by date when the experiment page is loaded.

yes, this is the default behaviour that the most recently modified one will be more on top.

**Review by jatinriverlane — APPROVED** [review]
Thanks @safeamiiir

**Pull request merged**